console/src/channel/services/ontology.tsx

```diff
@@ -30,7 +34,7 @@ import { type Ontology } from "@/ontology";
 import { useConfirmDelete } from "@/ontology/hooks";
 import { Range } from "@/range";
 import { Schematic } from "@/schematic";
-
+import {isCalculated} from "@synnaxlabs/client/src/channel/client";
```

Comment:
This import will not work. You need to import it from `@synnaxlabs/client`. This will only work in dev, but will break in production.

Reply:
This still has not been fixed